### PR TITLE
add support for multi language

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
@KaitCrawford ready for review :)

File: molo/surveys/tests/test_views.py [hunk 1]
```diff
@@ -239,23 +247,42 @@ def test_multi_step_multi_submissions_anonymous(self):
...
-        response = self.client.get(self.english.url)
-
+            self.create_molo_survey_page(parent=self.surveys_index)
+        response = self.client.get("/")
         self.assertContains(response,
                             '<a href="{0}">Take The Survey</a>'.format(
                                 molo_survey_page.url))
         self.assertContains(response, molo_survey_page.intro)
 
+    def test_tranlated_survey(self):
```
Comment (KaitCrawford) [inline molo/surveys/tests/test_views.py]: spelling error. should be `test_translated_survey`

File: molo/surveys/tests/test_views.py [hunk 2]
```diff
@@ -239,23 +247,42 @@ def test_multi_step_multi_submissions_anonymous(self):
...
+        self.user = self.login()
+        molo_survey_page, molo_survey_form_field = \
+            self.create_molo_survey_page(parent=self.surveys_index)
+
+        self.client.post(reverse(
+            'add_translation', args=[molo_survey_page.id, 'fr']))
+        tranlated_survey = MoloSurveyPage.objects.get(
+            slug='french-translation-of-test-survey')
+        tranlated_survey.save_revision().publish()
```
Comment (KaitCrawford) [inline molo/surveys/tests/test_views.py]: Spelling on the above two lines again :)

**Comment by KaitCrawford** [comment]
@saeedmb8, I left comments for spelling mistakes :) :+1: once those are fixed and you've added that other test

**Comment by KaitCrawford** [comment]
:+1: :)

**Pull request merged**